plugins/bulk_upload/xml/batch/BulkUploadEngineXml.php

```diff
@@ -608,7 +608,11 @@ protected function handleItemUpdate(SimpleXMLElement $item)
 				$resource = null;
 			}
 		}
-		
+
+		$keepManualThumbnails = false;
+		if(isset($item->keepManualThumbnails) && $item->keepManualThumbnails == 1)
```

Comment:
keepManualThumbnails  should be string, when value is strtolow(true) then set it